Pods/Kingfisher/Sources/Cache/DiskStorage.swift

```diff
@@ -196,11 +196,18 @@ public enum DiskStorage {
         }
 
         func cacheFileName(forKey key: String) -> String {
```

Comment:
Make sure to rerun pod install once you change back to the old versions (except for FutureNova) so these changes don't appear anymore.